src/themes/device.ts

```diff
@@ -25,7 +25,7 @@ const device = Object.freeze({
     tabletM: `(min-width: ${size.mobileL}px) and (max-width: ${size.tabletL}px)`,
     tabletS: `(max-width: ${size.tabletS}px)`,
     tablet: `(max-width: ${size.tablet}px)`,
-    tabletL: `(max-width: ${size.tabletL}px)`,
+    tabletL: `(max-width: ${size.tabletL - 1}px)`,
```

Comment:
Won't this affect alot of UI from this size?

Reply:
you're right, this has side-effects. but we're gonna introduce new object for the useMediaQuery. I'll revert this back to normal. thanks man @prince-deriv :pray: